bootstrap/ansible/playbook.yml

```diff
@@ -390,10 +400,18 @@
     # Development only provisioning tasks.
     - name: Run Development Provisioning Tasks
       block:
-        # Don't do anything as there aren't any development-specific tasks yet.
-        - meta: noop
-      # Uncomment when tasks are added.
-      # when: provisioning_tasks == true and deployment == "dev"
+
+        - name: Install vim
+          yum:
+            name: vim
+            state: present
+
+        - name: Modify .bashrc to start in application directory by default
```

Comment:
Given that this is here, you can remove the recommendation in `README.md` to manually add it to the file.